src/App.js

```diff
@@ -45,25 +45,32 @@ export const personas = [
 const FullScreenGrid = styled(Grid)`
   height: 100lvh;
   width: ${({ showSelector }) =>
-    showSelector ? 'calc(100lvw - 430px)' : '100lvw'};
-  margin-left: ${({ showSelector }) => (showSelector ? '430px' : '0px')};
+    showSelector ? 'calc(100lvw - 450x)' : '100lvw'};
+  margin-left: ${({ showSelector }) => (showSelector ? '450px' : '0px')};
   overflow: hidden;
   box-sizing: border-box;
+  background-color: ${({ showSelector, color }) =>
+    showSelector ? color : 'white'};
   transition:
+    background-color ease 0.1s,
     width ease-in-out 0.7s,
+    padding ease-in-out 0.7s,
     margin-left ease-in-out 0.7s;
-  border: ${({ showSelector, color }) =>
-    showSelector ? '20px solid ' + color : 'none'};
+  padding: ${({ showSelector }) => (showSelector ? '20px' : '0px')};
+
+  &:before {
+    content: '';
+  }
```

Comment:
Mmm... to be fair not sure if I understand correctly, but to me it looks like it might not worth it to implement. It would improve, but I think we should take losses and gains of this demonstrator so we can improve the following one.

P.S thanks for the flag up. I've removed `before` was a left over from other implementation, but either way I've had to update other components as they did not have background so it was taking from the parent element